tests/checks/tmux-complete.fish

```diff
@@ -102,3 +102,15 @@ isolated-tmux send-keys C-e M-f Space nothing
 tmux-sleep
 isolated-tmux capture-pane -p
 # CHECK: prompt 7> echo suggest nothing
+
+isolated-tmux send-keys C-u 'bind \cs forward-char-passive' Enter C-l
+isolated-tmux send-keys C-u 'bind \cp backward-char' Enter C-l
+isolated-tmux send-keys C-u 'echo do not accept this' Enter C-l
+tmux-sleep
+isolated-tmux send-keys 'echo do not accept thi' C-p C-p Delete C-p C-s 'h'
+tmux-sleep
+isolated-tmux send-keys C-s C-s C-s 'x'
+tmux-sleep
+isolated-tmux capture-pane -p
+tmux-sleep
```

Comment:
the first and last sleep are unnecessary because there is no relevant async work going on.
Also, I'd `C-b` instead of `C-p` would be less confusing